frontend/public/components/masthead-toolbar.jsx

```diff
@@ -238,13 +265,21 @@ class MastheadToolbar_ extends React.Component {
 
   render() {
     const { isApplicationLauncherDropdownOpen, isHelpDropdownOpen, showAboutModal } = this.state;
-    const { flags } = this.props;
+    const { flags, activeNamespace } = this.props;
     const resources = [{
       kind: clusterVersionReference,
       name: 'version',
       isList: false,
       prop: 'obj',
     }];
+
+    let serviceMeshItem;
+    if (activeNamespace && activeNamespace !== ALL_NAMESPACES_KEY) {
+      serviceMeshItem = <Firehose key="firehose" resources={[{kind: 'Namespace', name: activeNamespace, prop: 'namespace', isList: false}]}>
+        <ServiceMeshItem key="serviceMeshItem" />
```

Comment:
Why do we need `key`?

Reply:
Removed it here, only needed it in `<DropdownItem ../>`'s

Reply:
Removed it here, only needed it in `<DropdownItem ../>`'s and `<Firehose ../>`